keras_trainer/data_generators.py

```diff
@@ -41,6 +42,7 @@ def set_processing_attrs(self,
                 to use for random transformations and normalization.
             random_crop_size: Size of the random crop. Either a percentage of the original image (0,1) that will do square
                 crop, a fixed size (tuple), or integer where the value will set equally to both dimensions.
+            custom_crop: If True, will crop images according to the manifest file.
```

Comment:
Correct me if I'm wrong, but we haven't mentioned the term "manifest files" in `keras-trainer` yet, so: 

```suggestion
            custom_crop: If True, will crop images according to the dataframe's crop coordinates information contained in `z_col`.
```